### PR TITLE
Mail holidays.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

This was  a silly idea we came up with.
On Sundays (determined by the date and time of the start of the round, as determined by the server), as well as certain mail holidays respected around the world, mail will not arrive that shift. Just like in real life.

While I was in the neighborhood I documented the code in holidays.dm for ease of use.

## Why It's Good For The Game

Full commitment to the bit, realism to the cold unerring machine that is the mailroom, and some days cargo will just have an easier job than others.

Holidays can be configured to be mail holidays too, for easier setup when adding a holiday to the game in the future.

## Changelog
:cl:
add: On Sundays and given mail holidays, mail will not arrive on station.
code: Documented code in the holidays file.
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/economy.dm [hunk 1]
```diff
@@ -47,9 +47,13 @@ SUBSYSTEM_DEF(economy)
 	var/import_total = 0
 	/// Number of mail items generated.
 	var/mail_waiting = 0
+	/// Mail Holiday: AKA does mail arrive today? Always blocked on Sundays.
+	var/mail_blocked = FALSE
 
 /datum/controller/subsystem/economy/Initialize(timeofday)
 	var/budget_to_hand_out = round(budget_pool / department_accounts.len)
+	if(time2text(world.timeofday, "DDD") == "Fri")
```
[the remark this answers is not in the document]
Comment (interestingusernam3) [reply code/controllers/subsystem/economy.dm:55]: לצחוק בקול רם

**Comment by Qustinnus** [comment]
is using real dates to decide when features are available really something that improves the game?

**Comment by Qustinnus** [comment]
inb4 "but holiday events exist"

**Comment by Valorium-RB** [comment]
Would this code also be able to set a precedent for special holiday-based mail? For example, getting costume pieces in the mail during Halloween or Valentine candies during Valentine's Day?

**Comment by Mothblocks** [comment]
thats cute but out of scope

**Comment by coiax** [comment]
> 
> 
> is using real dates to decide when features are available really something that improves the game?
image

yes.

**Comment by Mothblocks** [comment]
> is using real dates to decide when features are available really something that improves the game?

its funny

**Pull request merged**